docker-compose.yaml

```diff
@@ -12,4 +12,5 @@ services:
       - .env
     volumes:
       - ./models:/models:cached
+      - /tmp/generated/images/:/tmp/generated/images/
```

Comment:
```suggestion
      - ./images/:/tmp/generated/images/
```

Reply:
so we keep it at the same level folder of the models

Reply:
@mudler yes , it looks better to Jeep it in the same folder